src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -371,34 +371,30 @@ public function thisProductHasDateTimeAttributeWithDate(ProductInterface $produc
      * @Given /^(this product) has option "([^"]+)" with values "([^"]+)" and "([^"]+)"$/
      * @Given /^(this product) has option "([^"]+)" with values "([^"]+)", "([^"]+)" and "([^"]+)"$/
      */
-    public function thisProductHasOptionWithValues(
-        ProductInterface $product,
-        $optionName,
-        $firstValue,
-        $secondValue,
-        $thirdValue = null
-    ) {
+    public function thisProductHasOptionWithValues(ProductInterface $product, $optionName, ...$values)
+    {
         /** @var ProductOptionInterface $option */
         $option = $this->productOptionFactory->createNew();
 
         $option->setName($optionName);
         $option->setCode(strtoupper($optionName));
 
-        $firstOptionValue = $this->addProductOption($option, $firstValue, 'POV1');
-        $secondOptionValue = $this->addProductOption($option, $secondValue, 'POV2');
+        $firstOptionValue = $this->addProductOption($option, $values[0], 'POV1');
+        $secondOptionValue = $this->addProductOption($option, $values[1], 'POV2');
```

Comment:
It is possible to use this step with only one value, so it is not a best solution. What with foreach with name generation? `POV` + index or something similar?